web/js/map-OpenLayers.js

```diff
@@ -29,6 +29,13 @@ function fixmystreet_update_pin(lonlat) {
             if ( lb.length === 0 ) { lb = $('#form_name').prev(); }
             lb.before(data.extra_name_info);
         }
+        // If the category filter appears on the map and the user has selected
+        // something from it, then pre-fill the category field in the report,
+        // if it's a value already present in the drop-down.
+        var category = $("#categories").val();
```

Comment:
Idly wondering if this should be filter_categories or something to prevent possible confusion over a different category drop down in future.